mix.exs

```diff
@@ -33,7 +33,10 @@ defmodule ExTealDirectUpload.MixProject do
       {:ex_teal, ">= 0.27.0"},
       {:ex_aws, "~> 2.5"},
       {:ex_aws_s3, "~> 2.5"},
+      {:ex_aws_sts, "~> 2.2"},
```

Comment:
I'm wondering if this works, if we can make this an optional dependency rather than a required one.